src/main/java/com/microsoft/sqlserver/jdbc/SQLServerLexer.java

```diff
@@ -20,7 +20,7 @@
 @SuppressWarnings({"all", "warnings", "unchecked", "unused", "cast"})
 class SQLServerLexer extends Lexer {
     static {
-        RuntimeMetaData.checkVersion("4.9.3", RuntimeMetaData.VERSION);
+        RuntimeMetaData.checkVersion("4.9.2", RuntimeMetaData.VERSION);

```

Comment:
why is this change necessary?

Reply:
It shouldn't be. Mostly likely just a merge conflict. That's not something I purposefully changed.

Reply:
This should be reverted.

Reply:
```suggestion
        RuntimeMetaData.checkVersion("4.9.3", RuntimeMetaData.VERSION);

```